specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json

```diff
@@ -82,6 +82,62 @@
         }
       }
     },
+    "/prebackup": {
+      "post": {
+        "tags": [
+          "PreFullBackup"
+        ],
+        "operationId": "PreFullBackup",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "description": "Pre-backup operation for checking whether the customer can perform a full backup operation.",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "preBackupOperationParameters",
+            "schema": {
+              "$ref": "#/definitions/PreBackupOperationParameters"
+            },
+            "description": "Optional parameters to validate prior to performing a full backup operation."
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Queued validating pre full backup operation",
+            "headers": {
+              "Retry-After": {
+                "description": "The recommended number of seconds to wait before calling the URI specified in Azure-AsyncOperation.",
+                "type": "integer",
+                "format": "int64"
+              },
+              "Azure-AsyncOperation": {
```

Comment:
This is not the correct method for long-running operations. See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#lro-returns-operation-location and https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations--jobs.

We could allow this for a prerelease, but it'll need to be fixed for GA. Please inform the certs team.